internal/argocd/cluster/cluster.go

```diff
@@ -0,0 +1,126 @@
+package cluster
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/redis/go-redis/v9"
+
+	"github.com/argoproj/argo-cd/v2/common"
+	appv1 "github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
+	cacheutil "github.com/argoproj/argo-cd/v2/util/cache"
+	appstatecache "github.com/argoproj/argo-cd/v2/util/cache/appstate"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	ConnectionStatusConnected    = "Connected"
+	ConnectionStatusDisconnected = "Disconnected"
+)
+
+/*
+UpdateClusterConnectionInfo updates cluster info with connection state and time in mapped cluster
+*/
```

Comment:
To be more in line with the rest of the code base, and with [Effective Go](https://go.dev/doc/effective_go#commentary), can you please use line comments instead of block comments for commentary of functions and other pieces of code?